cli/command/context/list_test.go

```diff
@@ -36,20 +35,6 @@ func TestList(t *testing.T) {
 	golden.Assert(t, cli.OutBuffer().String(), "list.golden")
 }
 
-func TestListNoContext(t *testing.T) {
```

Comment:
Why is this test removed ?

Reply:
Since the code handling the default context in the list command code has been removed this test did not provide additional coverage.

Reply:
:+1